code/game/g_main.c

```diff
@@ -2782,7 +2782,7 @@ void CheckVote( void ) {
 			}
 		}
 	}
-	if ( !level.voteTime ) {
+	if ( !level.voteTime || level.unpauseTime > level.time ) {
```

Comment:
Voting during pause was enabled on purpose. It looks like a vote won't be able to pass during pause after this change?

Reply:
Yeah ... I thought you forgot it, but if that was enabled on purpose, ignore it ...